synapse/models/inet.py

```diff
@@ -1489,6 +1496,71 @@ def getModelDefs(self):
                     ('inet:ssl:jarmsample', ('comp', {'fields': (('server', 'inet:server'), ('jarmhash', 'inet:ssl:jarmhash'))}), {
                         'doc': 'A JARM hash sample taken from a server.'}),
 
+                    ('inet:service:platform', ('guid', {}), {
+                        'doc': 'A network platform which provides services.'}),
+
+                    ('inet:service:instance', ('guid', {}), {
+                        'doc': 'An instance of the platform such as Slack or Discord instances.'}),
+
+                    ('inet:service:object:status', ('int', {'enums': svcobjstatus}), {
+                        'doc': 'An object status enumeration.'}),
+
+                    ('inet:service:account', ('guid', {}), {
+                        'doc': 'An account within a service platform. Accounts may be instance specific.'}),
+
+                    ('inet:service:permission:type:taxonomy', ('taxonomy', {}), {
+                        'interfaces': ('meta:taxonomy',),
+                        'doc': 'A permission type taxonomy.'}),
+
+                    ('inet:service:permission', ('guid', {}), {
+                        'interfaces': ('inet:service:object',),
+                        'doc': 'A permission which may be granted to a service account or role.'}),
+
+                    ('inet:service:rule', ('guid', {}), {
+                        'interfaces': ('inet:service:object',),
+                        'doc': 'A rule which grants or denies a permission to a service account or role.'}),
+
+                    ('inet:service:login', ('guid', {}), {
+                        'interfaces': ('inet:service:action',)}),
+
+                    ('inet:service:login:method:taxonomy', ('taxonomy', {}), {
+                        'interfaces': ('meta:taxonomy',),
+                        'doc': 'A taxonomy of inet service methods.'}),
+
+                    ('inet:service:login:type:taxonomy', ('taxonomy', {}), {
```

Comment:
This taxonomy type also doesn't have a form and doesn't appear to be used yet

Reply:
thanks on both! :+1: